guides/source/active_record_validations.md

```diff
@@ -1269,8 +1269,10 @@ class Person < ApplicationRecord
 end
 ```
 
-As shown in the example, you can also combine standard validations with your
-own custom validators.
+Custom validator classes are conventionally stored in `app/validators` (e.g. 
```

Comment:
`app/validators` works but it isn't technically relevant here. They would be also autoloaded if they where in `app/models` or even `app/controllers` They are just autoloaded by the name, no matter which folder they are. Maybe we should tell people about that instead of telling about a folder that isn't common in Rails applications (neither default)?

Reply:
Thanks! We should probably revert then. I swear I tried another directory name and it failed to auto load so I made the false assumption of assuming it was explicitly defined

Reply:
Sorry, for overlooking this @rafaelfranca @searls